<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>REPUESTOS</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(22, 71, 115);</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>761</width>
     <height>501</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>FiraCode Nerd Font</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(180, 190, 201);</string>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>CATALOGO DE REPUESTOS</string>
    </attribute>
    <widget class="QLineEdit" name="txtCodeName">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>741</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>FiraCode Nerd Font</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 8px;
border: 1px solid;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>CODIGO O NOMBRE</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbMotoL">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>80</y>
       <width>321</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>FiraCode Nerd Font</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(32, 32, 34);</string>
     </property>
     <property name="maxVisibleItems">
      <number>50</number>
     </property>
     <property name="placeholderText">
      <string>SELECCIONAR MOTO</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbCategoryL">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>251</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>FiraCode Nerd Font</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(32, 32, 34);</string>
     </property>
     <property name="maxVisibleItems">
      <number>50</number>
     </property>
     <property name="placeholderText">
      <string>SELECCIONAR LA CATEGORIA</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="tRepuestosL">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>741</width>
       <height>261</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>FiraCode Nerd Font</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSearch">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>140</y>
       <width>301</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>FiraCode Nerd Font</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(32, 32, 34);</string>
     </property>
     <property name="text">
      <string>BUSCAR</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnClear">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>FiraCode Nerd Font</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(32, 32, 34);</string>
     </property>
     <property name="text">
      <string>LIMPIAR</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbModelo">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>80</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>FiraCode Nerd Font</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(32, 32, 34);</string>
     </property>
     <property name="maxVisibleItems">
      <number>50</number>
     </property>
     <property name="placeholderText">
      <string>MODELO</string>
     </property>
     <item>
      <property name="text">
       <string>2000</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2001</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2002</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2003</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2004</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2005</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2006</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2007</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2008</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2009</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2010</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2011</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2012</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2013</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2014</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2015</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2016</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2017</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2018</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2019</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2020</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2021</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2022</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2023</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2024</string>
      </property>
     </item>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnReturn">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>540</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>FiraCode Nerd Font</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(32, 32, 34);</string>
   </property>
   <property name="text">
    <string>SALIR</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
